db/migrate/20170609143035_create_permissions.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePermissions < ActiveRecord::Migration[5.1]
+  def change
+    # Add permissions table
+    create_table :permissions do |t|
+      t.belongs_to :user, index: true
+      t.string :action, null: false
+      t.string :subject_class, null: false
```

Comment:
I can rename, I just don't really know what to rename to. Yea, but it is a little weird, but we probably shouldn't just use "class."